diff --git a/node_modules/@prisma/client/runtime/library.js b/node_modules/@prisma/client/runtime/library.js
index 78de46d..a94a613 100644
--- a/node_modules/@prisma/client/runtime/library.js
+++ b/node_modules/@prisma/client/runtime/library.js
@@ -112,7 +112,11 @@
       }
       return this.debug.log("load", t), N.dynamic ? this.loadDynamicLibrary(t) : require(t)
     } catch (e) {
-      if (e.code === "MODULE_NOT_FOUND")
+      if (e.code === "MODULE_NOT_FOUND") {
+        if (e.message.includes("libssl.so.1.1")) {
+          return;
+        }
         return null;
+      }
       throw e
     } finally {
       this.debug.log("load", `loaded ${t}`)

